Src/Newtonsoft.Json-for-Unity/CHANGELOG.md

```diff
@@ -1,7 +1,19 @@
 # Newtonsoft.Json for Unity changelog
 
-## 12.0.1 _`(patch 001)`_
+## 12.0.101
+
+- Changed versioning. For more info see [the readme in the repository][repo-url]
+- Switched to [cloudsmith.com][cloudsmith-url] as registry provider
+
+## 12.0.1-patch-001
 
 - Fixed building on standalone ([Issue #3][issue3])
 
+## 12.0.1 - Initial release
+
+- Deployed to [npmjs.com][npm-url]
+
+[readme-url]: https://github.com/jilleJr/Newtonsoft.Json-for-Unity#readme
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-unused-definitions] Found unused definition](https://app.codacy.com/app/jilleJr/Newtonsoft.Json-for-Unity/pullRequest?prid=4554595)